conf/modules.config

```diff
@@ -26,8 +26,8 @@ process {
         ]
     }
 
-    withName: MULTIQC {
-        ext.args   = params.multiqc_title ? "--title \"$params.multiqc_title\"" : ''
+    withName: 'MULTIQC' {
```

Comment:
Why single quotation marks? Because new template wants them?

Reply:
It's template :shrug: